modules/manage/pages/kubernetes/security/tls/use-cert-manager.adoc

```diff
@@ -0,0 +1,412 @@
+= Use cert-manager to manage TLS certificates
+:description: Enable TLS encryption in your Redpanda cluster and use cert-manager to simplify the process of obtaining, renewing, and using certificates.
+
+Learn how to enable TLS encryption in your Redpanda cluster and use glossterm:cert-manager[^] to simplify the process of obtaining, renewing, and using certificates.
```

Comment:
This first line sounds more like a description statement than an intro. I think you could remove it and just add the gloss term to the next line.